### PR TITLE
huawei honor 5x

### PR DESCRIPTION
**Pull request by tshering1980**

### CODE REVIEW

File: manifests/huawei_kiwi.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<project path="device/huawei/kiwi" name="android_device_huawei_kiwi" remote="los" />
```
Comment (jbruechert) [inline manifests/huawei_kiwi.xml]: Please fix the indentation here. (Have a look at the other manifests for an example)

Comment (tshering1980) [reply manifests/huawei_kiwi.xml]: please have a look

Comment (tshering1980) [reply manifests/huawei_kiwi.xml]: Indentation fixed

**Comment by tshering1980** [comment]
Indentation updated

**Pull request merged**